lib/Sema/SemaDecl.cpp

```diff
@@ -8235,6 +8235,10 @@ void Sema::CheckMain(FunctionDecl* FD, const DeclSpec& DS) {
           Context.hasSameType(QualType(qs.strip(PT->getPointeeType()), 0),
                               Context.CharTy)) {
         qs.removeConst();
+
+        if (Context.getDefaultAS() == qs.getAddressSpace()) {
+          qs.removeAddressSpace();
+        }
```

Comment:
> Remove braces (LLVM coding style).

This:
http://llvm.org/docs/CodingStandards.html

is the LLVM coding style rules, right?

It does not appear to address the issue of braces on single-line blocks? (although I might be failing to see the very obvious thing right in front of me again). One would expect it to be lurking under "microscopic details":
http://llvm.org/docs/CodingStandards.html#microscopic-details

> Please add a test.

Of course.